_posts/2021-04-21-unstructured-knowledge-sharing.markdown

```diff
@@ -0,0 +1,73 @@
+
+(example screenshot)
+
+One other important thing to say about the #dev-help channel: as engineers it's tempting to solve problems with new and novel tooling, but in the case of #dev-help we found a way to use our existing tooling in a slightly different way. Sometimes the best solution is one you already have. We could have investigated dedicated software like a StackOverflow for Teams, or we could have built our own knowledge-base tool. Instead we put some rules around how we would use a Slack channel, and gained a huge benefit at a fraction of the cost.
+
+## Internal Live-Streaming
+
+Occasionally an Artsy engineer will broadcast that they're taking on work that is ripe for knowledge-sharing, and they'll spin up a Zoom call for others to join. It might be something they know how to solve — like when [Roop][roop] spun up a call to walk through how he'd use [Observable][observablehq] to explore search index weighting. There might be more uncertainty to the problem, like when [Pavlos][pavlos] started up a call to investigate a CI build failure using [`git bisect`][git-bisect]. It might be somewhere in between, like when [Adam][adam-b] was doing some refactoring of our React Native navigation. 
+
+Sometimes this might end up looking more like a pairing or mobbing session, but often it ends up looking more like live-streaming. An engineer is demonstrating by solving a real problem. If audience members can contribute, great — but they are also free to tag along and learn.
+
+## Pairing
+
+[Yuki][yuki] wrote on this blog [about pair-programming at Artsy][yukis-article]. Pairing isn't an unusual practice for development teams....but there is something notable in regards to knowledge-sharing. 
+
+We're somewhere in the middle on the pairing frequency spectrum at Artsy. We don't have dedicated pairs working on problems together all day/every day. But we also don't only pair when we're stuck. 
+
+Pairing when you're stuck is great, and it can help move you forward on a problem. If that's the only time you pair, though, you're missing out on a massive learning opportunity. 
+
+Pairing to build a feature gives you exposure to an entire toolbox you might never have used before. How does your pair approach a problem? How do they manage their time? What development tools do they use? What techniques do they use for testing and debugging? How do they tighten their feedback loop? This is knowledge-sharing beyond the code or the product you're building — it's knowledge-sharing of tools, skills, and habits.
+
+---
+
+What are the unstructured/unscheduled ways in which your team shares knowledge? Leave us a note in the comments!
+
+[ash]: https://twitter.com/ashfurrow
+[knowledge-shares]: https://artsy.github.io/blog/2020/12/09/share-your-knowledge/
+[roop]: https://github.com/anandaroop
```

Comment:
@anandaroop would you prefer a different URL here?

Reply:
Works for me 👍🏽